pkgs/development/python-modules/vaa/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+  #build
+, flit
+
+  # tests
+, pytestCheckHook
+, cerberus
+, django
+, djangorestframework
+, marshmallow
+, pyschemes
+, wtforms
+, email_validator
+
+}:
+
+buildPythonPackage rec {
+  pname = "vaa";
+  version = "0.2.1";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "life4";
+    repo = pname;
+    rev = "refs/tags/v.${version}";
+    hash = "sha256-24GTTJSZ55ejyHoWP1/S3DLTKvOolAJr9UhWoOm84CU=";
+  };
+
+  nativeBuildInputs = [
+    flit
```

Comment:
We should patch `pyproject.toml` to use
```
[build-system]
requires = ["flit_core"]
build-backend = "flit_core.buildapi"
```

Reply:
done :+1: 
Side question: Why prefer `flit_core` over `flit`?